mobile/src/calendar/calendar-actions-button-group.tsx

```diff
@@ -32,16 +36,26 @@ export class ActionsButtonGroupImpl extends Component<ActionButtonGroupProps & A
 
         return (
             <View style={calendarActionsStyles.container}>
-                <VacationActionButton interval={intervalsBySingleDaySelection.vacation} disabled={this.props.disableActionButtons} />
+                <VacationActionButton
```

Comment:
do we check that user can't change dates in approved vacation? Only cancel it.

Reply:
yes, we do, but I thought to add it with edit next PR

Reply:
alright

Reply:
do you want to add a separate dialog for that, perhaps?

Reply:
yes